docs/release-notes/8.9.asciidoc

```diff
@@ -1,6 +1,31 @@
 [[release-notes-header-8.9.0]]
 == 8.9
 
+[discrete]
+[[release-notes-8.9.1]]
+=== 8.9.1
+
+[discrete]
+[[breaking-changes-8.9.1]]
+==== Breaking changes
+
+There are no breaking changes in 8.9.1.
+
+[discrete]
+[[enhancements-8.9.1]]
+==== Enhancements
+* Event correlation rules now support negative correlation. 
+
+[discrete]
+[[bug-fixes-8.9.1]]
+==== Bug fixes 
+
+* Fixes a copy to clipboard bug that affected non-ECS fields ({pull}162883[#162883]).
+* Fixes rounding issues in the *Top alerts by* table on the Alerts page ({pull}162647[#162647]).
+* Fixes bug that prevented controls from being rendered on {elastic-sec} dashboards ({pull}162514[#162514]).
+* Fixes exceptions in investigate in timeline ({pull}162190[#162190]).
```

Comment:
@kqualters-elastic after reading the description, I'm still not quite sure what the bug (or bugs) were. Was this bug related to the "open field in Timeline" action that users could take from anywhere within the Security app?

Reply:
I think this is a little too vague, maybe something like ""Fixes a bug where an alert based on a rule with exceptions could cause timeline to fail to load properly" ?

Reply:
might have to scrap this all together, as the pr has been merged to main but the backport is being blocked by unrelated test failures https://github.com/elastic/kibana/pull/163373

Reply:
Ok, let me know if I need to remove it from the release notes - thank you!

Reply:
You can remove it @nastasha-solomon, thanks for checking!

Reply:
```suggestion
```